appinventor/components/src/com/google/appinventor/components/common/YaVersion.java

```diff
@@ -938,11 +939,12 @@ private YaVersion() {
   //   TwitPic. The TwitPic_API_Key property is now deprecated and hidden.
   public static final int TWITTER_COMPONENT_VERSION = 4;
 
-  // For VERTICALARRANGEMENT_COMPONENT_VERSION 2:
+  // For VERTICALARRANGEMENT_COMPONENT_VERSION 3:
```

Comment:
Also, you need to add lines earlier in the file that say

  // For YOUNG_ANDROID_VERSION ???:
  // - HORIZONTALARRANGEMENT_COMPONENT_VERSION was incremented to 4.
  // - VERTICALARRANGEMENT_COMPONENT_VERSION was incremented to 4